modules/ROOT/nav.adoc

```diff
@@ -89,7 +89,8 @@
  ** xref:cassandra/cassandra-connector-studio.adoc[Cassandra Studio Configuration]
  ** xref:cassandra/cassandra-connector-config-topics.adoc[Cassandra Additional Configuration]
  ** xref:cassandra/cassandra-connector-xml-maven.adoc[Cassandra XML and Maven Support]
- ** xref:cassandra/cassandra-connector-examples.adoc[Cassandra Examples]
+ ** xref:cassandra/cassandra-connector-examples.adoc[Example for Creating a Cassandra Database Table]
```

Comment:
The nav looks incorrect. Why isn't the Example for creating a Cassandra Database table part of the Examples topic?